I2C.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
If there is going to be separate documentation for each class, we should probably have a separate directory